latex/epoch.tex

```diff
@@ -231,6 +226,146 @@ \subsection{Accounting Functions}
   \label{fig:functions:epoch}
 \end{figure}
 
+%%
+%% Figure - Functions for the Reward Calculation
+%%
+\begin{figure}[htb]
+  \begin{align*}
+      & \fun{maxPool} \in \PrtclConsts \to \Coin \to [0, 1] \to [0, 1] \to \Coin
+      & \text{max pool reward} \\
+      & \fun{maxPool}~\var{pc}~\var{R}~\sigma~\var{s}~\var{poolConsts} = \\
+      & ~~~\floor*{
+           \frac{R}{1 + a_0}
+           \cdot
+           \left(\sigma' + s'\cdot a_0\cdot\frac{\sigma' - s'\frac{z_0-\sigma'}{z_0}}{z_0}\right)} \\
+      & ~~~\where \\
+      & ~~~~~~~(a_0, k) = \fun{maxPoolConsts}~pc \\
+      & ~~~~~~~z_0 = 1/k \\
```

Comment:
Is maxPoolConsts and poolConsts defined anywhere?

Reply:
nope, good catch!